### ISSUE TITLE
Synchro express II doesn't load in Blitter level 1 and 2

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#287]
Amiga Synchro Express II.adf

Blitter level 0:

<img width="910" alt="Bildschirmfoto 2020-02-20 um 22 41 08" src="https://user-images.githubusercontent.com/12561945/74981319-589cf200-5432-11ea-9b0a-6e4477e59acf.png">

Blitter level 1 and 2 (hangs):

<img width="910" alt="Bildschirmfoto 2020-02-20 um 22 41 46" src="https://user-images.githubusercontent.com/12561945/74981339-5fc40000-5432-11ea-940c-ff28eb0a7619.png">

Might be related to #179

**Comment by dirkwhoffmann** [comment]
Reason: Blitter DMA is switched off while the Blitter is operating.

```
[995] ( 77,  1) C156D4  0 BC-SDA 61AE 1040 Agnus: pokeDMACON(8040)
[995] ( 77,  3) C156D4  0 BC-SDA 61AE 1040 Agnus: setDMACON(3b3, 8040)
[995] ( 77,  3) C156D4  0 BCBSDA 61AE 1040 Agnus: Blitter DMA switched on
BLITTER Blit 413 (32,187) (0101) (0 0 78 0) 4e 2320 1589c 2060 
[995] (214, 73) C156F8  0 BCBSDA 61EC 1000 Agnus: pokeDMACON(40)
[995] (214, 75) C156F8  0 BCBSDA 61EC 1000 Agnus: setDMACON(3f3, 40)
[995] (214, 75) C156F8  0 BC-SDA 61EC 1000 Agnus: Blitter DMA switched off

```

In such occasions the Blitter freezes until the bus is available again (just as the Copper does). This is what vAmiga does 🙄:

```
    // Blitter DMA
    if (oldBLTEN ^ newBLTEN) {
        debug(DMA_DEBUG, "Blitter DMA switched %s\n", newBLTEN ? "on" : "off");

        if (!newBLTEN) {
            blitter.kill();
        }
    }
```

Next step: Write DMA on / off test cases for the Blitter

**Comment by dirkwhoffmann** [comment]
Fixed in v0.60.1
